man/usr/share/man/man8/piadm.8.md

```diff
@@ -326,6 +326,16 @@ piadm(8) -- Manage SmartOS Platform Images
  [root@smartos ~]#
 ```
 
+## ENVIRONMENT VARIABLES
+
+    The following environment variables affect the execution of PIADM(8).
```

Comment:
1.) Lowercase to `piadm(8)`.

2.) I'd HIGHLY recommend using `SUM` instead of `MD5SUM` for the variable names in case we opt to change MD5 out later. I'll talk about this a bit more in the code itself, but this place is OBVIOUS to not commit to a specific algorithm.

3.) Add an explicit sentence or two.  Here's my suggestion (ignore/edit/use as you see fit):

`These variables only affect images piadm(8) downloads internally.`

4.) The code may make it more clear, but if I utter:

`piadm install https://kebe.com/~danmcd/webrevs/platform-OS-xxxx.tgz`

will I have to use `PIADM_NO_SUM` (assuming you take my advice in 2) to not have it go searching for an md5sums.txt file?

Reply:
That answers 4.

1 hasn't been fixed yet, neither has 3.

2 is moving along nicely, modulo some nits I might find.